Lib/fontTools/varLib/featureVars.py

```diff
@@ -14,7 +14,9 @@
 from .errors import VarLibError, VarLibValidationError
 
 
-def addFeatureVariations(font, conditionalSubstitutions, featureTag="rvrn"):
+def addFeatureVariations(
+    font, conditionalSubstitutions, featureTag="rvrn", processLast=None
```

Comment:
this new `processLast` parameter does not seem to be used anywhere, I think it's ok if remove it altogether and automatically append when featureTag != "rvrn", otherwise insert at index 0 for "rvrn". I don't see its utility, why would one ever want to _not_ insert at 0 for "rvrn" or, conversely, to insert at 0 (instead of append) for any other feature tag that is != "rvrn".
Right now we ignore the `processing` attribute of DS rules if there is a `featureVarsFeatureTag` key in the designspace lib, since the two methods were intended to be alternative to one another:

https://github.com/fonttools/fonttools/blob/f026853cb28df60006347cecf69f57e76c56d91e/Lib/fontTools/varLib/__init__.py#L55-L56

I mean, we could chose to apply the rules `processing` attribute whenever present and pass it down to featureVars.addFeatureVariations. Only in this case it would make sense to keep the `processLast` parameter for addFeatureVariations function (though I am not 100% sure how useful that would be)

Reply:
> I mean, we could chose to apply the rules processing attribute whenever present and pass it down to featureVars.addFeatureVariations. Only in this case it would make sense to keep the processLast parameter for addFeatureVariations function (though I am not 100% sure how useful that would be)

That was my intention, to wire it up.

Reply:
Wired up now.

Reply:
I think I better understand your point about not exposing this parameter now. Let me think.

Reply:
Removed.